docs/source/installation.rst

```diff
@@ -7,12 +7,25 @@ Installation
     :local:
     :depth: 1
 
-Install from Conda
-------------------
+Install from Conda-Forge (suggested)
+------------------------------------
 
 .. warning::
+  The Raven-WPS conda package is not yet available on conda-forge. This documentation will be updated when it is.
 
-   TODO: Prepare Conda package.
+Create an Anaconda environment named `raven`:
```

Comment:
The naming of the venv in the following is not consistent (raven vs raven-env). I suggest "ravenwps-env"